frontend/src/css/app.sass

```diff
@@ -100,6 +100,46 @@ p, div
   max-width: 50%
   text-align: center
 
+.batch-send
```

Comment:
*Arbitrary comment location*

When I add multiple sends with amounts >1000 DAI I get a bunch of errors on input:

![multi-dai-batch-send-error](https://github.com/ScopeLift/umbra-protocol/assets/1479802/d50a1adf-a3a5-4bc8-ae79-12fad79e9474)

Reply:
Fixed in https://github.com/ScopeLift/umbra-protocol/commit/16d401071503c3b1fd70d820b2f16f0cb2a2bea8